packages/core/expressions/package.json

```diff
@@ -66,8 +66,8 @@
     "errorLimit": "2048KB"
   },
   "peerDependencies": {
-    "@kong/atc-router": "1.6.0-rc.1",
-    "@kong/kongponents": "9.0.0-alpha.105",
+    "@kong/atc-router": "^1.6.0-rc.1",
+    "@kong/kongponents": "^9.0.0-alpha.105",
```

Comment:
**Comment**: I noticed that some Kongponents peer dependencies use different versions than the majority.

Reply:
@portikM Do you know why in the remaining diff in this PR, the two Kongponents peers are not requiring the same versions as all the other packages?

Reply:
I think these two just got missed. I usually do search and replace and these two didn't meet the search criteria like the majority. Update them both in your PR please

Reply:
Will do.

That being said, it looks like CI can’t run PRs from a fork (I don’t have push permissions here).

Reply:
Oh, yeah, I didn't realize this is a fork. Any particular reason why this needs to be a fork? I know @adamdehaven was planning on doing something to make forking possible in this repo but I'm not sure what's the status there

Reply:
See this thread: https://kongstrong.slack.com/archives/CQK8J4VN3/p1713444926254169

Next week I need to update the CI as it currently requires secrets in order to run the tests, etc. You'll need to clone the repo and submit a PR from a branch for now

Reply:
@kleinfreund i'll grant write permissions for now

Reply:
You should be good for now; I'll get this sorted out in the next week when I'm back